### DISCUSSION TITLE
Flash.nvim preventing which-key from firing?

### DISCUSSION THREAD

**Discussion by megalithic** [folke/flash.nvim#30] [Q&A]
I haven't nailed down why/what, just yet, other than uninstalling flash.nvim and seeing which-key.nvim triggering again. I'll see what a bare bones config of which-key looks like, too, but, I wanted to go ahead and get this question out there, in case others have hit similar.

TL;DR -
which-key no longer triggers with my leader key `,`; i suspect it has somethign to do with the repeating previous jump with `,`, so it might just be that I have to disable that and utilize the fF/tT repeat usage instead.

**Comment by folke** [comment]
That will be it for sure. You can configure flash not to map to `,`. See the config section of the readme.
